Mathlib/Topology/Separation.lean

```diff
@@ -1045,6 +1045,16 @@ theorem r1Space_iff_inseparable_or_disjoint_nhds {X : Type*} [TopologicalSpace X
   ⟨fun _h x y ↦ (specializes_or_disjoint_nhds x y).imp_left Specializes.inseparable, fun h ↦
     ⟨fun x y ↦ (h x y).imp_left Inseparable.specializes⟩⟩
 
+theorem inseparable_of_nhds_neBot {x y : X} (h : NeBot (𝓝 x ⊓ 𝓝 y)) :
+    Inseparable x y :=
+  (r1Space_iff_inseparable_or_disjoint_nhds.mp ‹_› _ _).resolve_right fun h' => h.ne h'.eq_bot
+
+/-- A weaker version of `tendsto_nhds_unique` for `R1Space`. -/
+theorem tendsto_nhds_unique_inseparable {X Y : Type*} [TopologicalSpace X] [R1Space X]
```

Comment:
Please reuse the available type variable(s). Also, do we use latin letters for spaces without structure (`Y` in this case)?

Reply:
> Also, do we use latin letters for spaces without structure (`Y` in this case)?

Yes, the rest of the file still does this.